about.html

```diff
@@ -9,8 +9,22 @@
+          <picture>
+            <source srcset="assets/about/member/member-Robin-Pilling@2x.webp" type="image/webp" />
```

Comment:
Check the availability about Robin Pilling. Minimize the functional changes.

Reply:
fixed